### PR TITLE
Release v0.5.2

### PR DESCRIPTION
**Pull request by cirrusasf**
## Description of this release

Correct nodata assignment issue in flood_map that led to inaccurate mask creation

### Developer checklist

- [x] Assigned a reviewer
  
- [x] Indicated the level of changes to this package by affixing one of these labels:
  * major -- Major changes to the API that may break current workflows
  * minor -- Minor changes to the API that do not break current workflows
  * patch -- Patches and bugfixes for the current version that do not break current workflows
  * bumpless -- Changes to documentation, CI/CD pipelines, etc. that don't affect the software's version
- [x] (If applicable) Updated the dependencies and indicated any downstream changes that are required
- [x] Updated `CHANGELOG.md`
- [x] Added/updated documentation for these changes
- [x] Added/updated tests for these changes

### Reviewer checklist

- [x] Have all dependencies been updated?
- [x] Is the level of changes labeled appropriately?
- [x] Are all the changes described appropriately in `CHANGELOG.md`?
- [x] Has the documentation been adequately updated?
- [ ] Are the tests adequate?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.5.2]
+
+### FIXED
+* Fix datatype bug in `asf-tools.flood_map` that caused `flood_mask` to be written incorrectly.
```
Comment (jhkennedy) [inline CHANGELOG.md:13]:
```suggestion
### Fixed
* Fixed incorrect datatype being set for `flood_mask` GeoTIFFs leading to missing nodata.
```

**Comment by forrestfwilliams** [comment]
Now that we've finished the testing changes, I think this ready for review. @jhkennedy it would be great if you could take another look at this.

**Comment by forrestfwilliams** [comment]
PR #186 has been create to revert the temporary test changes we made to work around the `gdalcompare` bug. It can be merged once the upstream fix is released (likely in GDAL 3.7).

**Review by jhkennedy — APPROVED** [review]

**Review by forrestfwilliams — APPROVED** [review]
LGTM!

**Pull request merged**